client/signup/config/flows.js

```diff
@@ -280,7 +280,7 @@ function filterFlowName( flowName ) {
 	return flowName;
 }
 
-function filterDestination( destination, dependencies, flowName ) {
+function filterDestination( destination ) {
 	return destination;
 }
 
```

Comment:
seems like we might be able to inline this and just say something like `filterDestination: identity` or `filterDestination: a => a` from inside the "class" definition